test/libsolidity/syntaxTests/pragma/future_version.sol

```diff
@@ -0,0 +1,3 @@
+pragma solidity 10.0.0;
+// ----
+// SyntaxError: (0-23): Source file requires different compiler version (current compiler is 0.4.25-develop.2018.7.23+commit.1dac6eff.Darwin.appleclang - note that nightly builds are considered to be strictly less than the released version
```

Comment:
I think this expected error isn't ideal, since it will change from build to build and for every new version.

Reply:
True and that reminds me there might have been another place to test these.

Reply:
Removed it for the moment.